src/data/javascript-quiz.ts

```diff
@@ -1561,6 +1561,17 @@ const javascriptQuiz = [
       "The toFixed() method converts a number to a string and rounds it to a specified number of decimals.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed",
   },
+  {
+    Question:
+      "In JavaScript, what should be the output of 3 < 2 < 1",
+    Answer: "true",
+    Distractor1: "false",
+    Distractor2: "syntax error",
+    Distractor3: "Nan",
+    Explanation:
+      "It will be true because first evaluate 2 is lo than 3 and returns false then evaluate 1 is bigger than false and returns true",
```

Comment:
```suggestion
      "JavaScript will first evaluate 3 < 2 which will return false. Then JavaScript will evaluate false < 1 which will return true because false is converted to the number 0. ",
```